### DISCUSSION TITLE
2 Errors about source code

### DISCUSSION THREAD

**Discussion by LJRFox** [rustdesk/rustdesk#4184] [Q&A]
1. 'generated_bridge.dart' CAN NOT find in these dart files:
   a. rustdesk/flutter/lib/models/model.dart
   b. rustdesk/flutter/lib/models/native_model.dart
   c. rustdesk/flutter/lib/models/platform_model.dart
   
  2. '_Audio' CAN NOT find in 'rustdesk/flutter/libs/desktop/pages/desktop_setting_page.dart'

**Comment by LJRFox** [comment]
as we know, the lastest UI on all platform are build with Flutter, BUT some files are missing in the source code , the path is 'rustdest/flutter/lib'. 

someone can help me or am I not understanding the code correctly.

**Comment by deep-soft** [comment]
generated_bridge.dart
is generated in:
[https://github.com/rustdesk/rustdesk/blob/master/.github/workflows/bridge.yml](https://github.com/rustdesk/rustdesk/blob/master/.github/workflows/bridge.yml)
```
# This yaml shares the build bridge steps with ci and nightly.
name: Build flutter-rust-bridge
# 2023-04-19 15:48:00+00:00
...

      - name: Run flutter rust bridge
        run: |
          ~/.cargo/bin/flutter_rust_bridge_codegen --rust-input ./src/flutter_ffi.rs --dart-output ./flutter/lib/generated_bridge.dart

...
```

**Reply by LJRFox** [reply]
Yeah, i got it. thanks a loooooooooooooooot!

**Comment by Moettazz** [comment]
pls how to installed or get it
